src/routes/accounts.ts

```diff
@@ -0,0 +1,158 @@
+import { Request, Response } from "express";
+import { connect, resetClient } from "../client";
+import { getDestinationWallet } from "../destination-wallet";
+import { Client, Payment, Wallet, xrpToDrops } from "xrpl";
+import { fundingWallet } from "../wallet";
+import { BigQuery } from "@google-cloud/bigquery";
+import { config } from "../config";
+import { getTicket } from "../ticket-queue";
+import rTracer from "cls-rtracer";
+
+export default async function (req: Request, res: Response) {
+  const client = await connect();
+
+  let account;
+
+  if (req.body.destination) {
+    try {
+      account = getDestinationWallet(req.body.destination);
+      console.log(`${rTracer.id()} | User-specified destination: ${account}`);
+    } catch {
+      return res.status(400).send({
+        error: "Invalid destination",
+      });
+    }
+  } else {
+    account = Wallet.generate();
+    console.log(`${rTracer.id()} | Generated new account: ${account.address}`);
+  }
+
+  let amount = config.XRP_AMOUNT;
```

Comment:
nit: I think `DEFAULT_XRP_AMOUNT` is a slightly better name for that global.

```suggestion
  let amount = config.DEFAULT_XRP_AMOUNT;
```